### DISCUSSION TITLE
component loader edit?

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#7181] [Ideas]
I finally got around to watching https://www.youtube.com/watch?v=QJeLNCn62dE, which involved  discussion of editing in the component loaders. Datatables (the library behind browse and edit) might be capable of that.

That would also set up some possibility of making "interesting" unintended changes and such, and of course would add a bunch of complexity to what's currently a very simple tool.

The workaround/current capability is download/delete/upload, which will probably always be the best approach for true bulk operations but isn't great for fixing one typo in an "extra" &etc.

**Comment by Jegelewicz** [comment]
Personally, I would LOVE the edit capability in the component loaders. 

> isn't great for fixing one typo in an "extra" &etc.

is the reason. It really sucks to need to download, change one letter, save and re-upload, but it isn't the end of the world. If the potential for messes and additional complexity exceeds my occasional frustration, then life will go on. However, IF we had an equal/but opposite UNLOADER for all of these loaders, maybe the messes could be more easily corrected by users?

**Comment by Jegelewicz** [comment]
I spent a lot of time doing this yesterday...
![image](https://user-images.githubusercontent.com/5725767/174140522-9f30d436-00ce-4e54-bf02-5b12c8add71e.png)

I'm going to put this on the AWG agenda, even if we aren't getting to it until July....

**Comment by AJLinn** [comment]
I'm not sure if this is the right issue to add this issue to, but maybe someone can point me at a more appropriate one.

One of our curatorial assistants has been checking their records in the Browse/Edit table by clicking on the collection_object_id link on the left.
<img width="1136" alt="Screenshot 2023-05-25 at 4 56 11 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/4c308dee-021d-45c7-b085-3f4af209b10c">

When she tries to edit the content in the attribute value fields, they do not expand like they do in the old form and in the data entry form. This makes it impossible to edit without copying and pasting into another doc, then pasting the edited info back into that field again. I suggested she instead edit in the flat table view in the meantime, but is that something that can be fixed?
<img width="804" alt="Screenshot 2023-05-25 at 4 56 47 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/69fcaaf6-4ed0-495a-8164-cc7b4c252d9e">

Again, sorry if this fits better as a comment on another existing issue. Maybe #6161 is more appropriate?

**Comment by dustymc** [comment]
@AJLinn I moved that to a dedicated issue, which I'm calling "resolved" here.
